handlers/mapcss/post.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const parseMapCSS = require('mapcss-parse').parse;
+const adaptRules = require('../../adapters/rules');
+const Boom = require('boom');
+const db = require('../../connection');
+const uuidv4 = require('uuid/v4');
+
+module.exports = async (r, h) => {
+    try {
+        const mapcss = r.payload;
+        const rulesMapCSS = parseMapCSS(mapcss);
+        
+        return h.response(rulesMapCSS).header('Content-Type', 'application/json').code(200);
```

Comment:
...this alone will always return a 200 w/what ever mapcss parses regardless if the parsed is not usable.

For example, I just tried sending along something the parser does not really understand yet

```
way >:sameTags node:tagged {
  throwWarning: tr("Nodes duplicating parent way tags");
}
```

and it responds

```"[{"geometry":"node","warning":"tr(Nodes duplicating parent way tags)"}]"```


... this is where validating the response comes in... See comment on in the routes file for more!